src/components/pico-controls.js

```diff
@@ -55,6 +55,20 @@ module.exports.Component = registerComponent('pico-controls', {
     this.onButtonTouchEnd = function (evt) { onButtonEvent(evt.detail.id, 'touchend', self, self.data.hand); };
     this.onButtonTouchStart = function (evt) { onButtonEvent(evt.detail.id, 'touchstart', self, self.data.hand); };
     this.bindMethods();
+
+    // Pico4, at least on Wolvic, needs a different rotation offset
+    // for the hands model when this component is used via
+    // hand-controls.
+    // Note: at the time of writing, the Pico Browser on Pico4 claims
+    // to use oculus controllers and does not use this component. It
+    // probably also applies the same offset an oculus controller has.
+    if (this.el.components['hand-controls']) {
```

Comment:
We shouldn't not couple hand-controls with pico-controls. This logic should be in `hand-controls`. We already have Vive specific stuff over there